internal/main

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash -e
 
+## The version of Python we're building with in Jenkins. Changing this will cause a new Python
+## to be installed on Jenkins using pyenv. Cleanup is left as a manual task.
+## It's highly recommended that this version stay in sync with the version that the Dockerfile
+## is based on so that devs and Jenkins build the same way.
+PYVERSION=3.6.13
```

Comment:
Is there any way to have the `Dockerfile` and this file use a common version?

Reply:
Maybe have the rax-docs script do the install of the python version in the dockerfile and the jenkinsfile?

Reply:
Talked to @zzantozz personally. There's a test, which I saw, that keeps these in sync, but it could be something to look at in the future.